Doppler.ContactPolicies.Data.Access/Entities/User.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Doppler.ContactPolicies.Data.Access.Entities
+{
+    public class User
```

Comment:
This class is different than `SubscribersList` and `ContactPoliciesSettings`, if I am not wrong, you are using this only for auxiliary purposes in `ContactPoliciesSettingsRepository`. 

In my opinion, having this in the same namespace as the other ones will generate confussion.

Reply:
Sorry, I was wrong. 

Resolving the conversation.

Reply:
The User class is used to filter the accountName that is passed to me as a parameter